src/style.css

```diff
@@ -1 +1,17 @@
-/* styles go here */
+html {
+  margin: 0;
+  font-family: "Times New Roman", Times, serif;
+}
+
+fieldset {
+  margin-top: 0;
+  margin-bottom: 20px;
+}
+
+.formula {
```

Comment:
I think this name should be changed to something better describing what we are trying to style here.